girder-dandi-archive/tox.ini

```diff
@@ -70,6 +70,8 @@ ignore =
     E123
     # whitespace before ':'
     E203,
+    # lowercase argument name
+    N803,
```

Comment:
I'd prefer we add `# noqa: N803` to the offending lines, rather than allowing a global exception for all our code.